.github/workflows/precompiled_binaries.yml

```diff
@@ -1,6 +1,6 @@
 on:
   push:
-    branches: [ main ]
+    branches: [ develop ]
```

Comment:
We want to generate the precompiled binaries only on new releases and we use the main branch as release branch